app/src/main/java/com/codingblocks/cbonlineapp/mycourse/MyCourseViewModel.kt

```diff
@@ -144,6 +148,23 @@ class MyCourseViewModel(
             .enqueue(request)
     }
 
+    fun getLeaderboard(crUid: String?) {
```

Comment:
Make this function return the livedata Instead of making a separate variable also use Iivedata scope instead of viewModel scope